service/src/main/java/uk/gov/hmcts/reform/fpl/validators/HasChildrenNameValidator.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import uk.gov.hmcts.reform.fpl.model.Child;
+import uk.gov.hmcts.reform.fpl.model.Children;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasChildName;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class HasChildrenNameValidator implements ConstraintValidator<HasChildName, Children> {
+
+    @Override
+    public void initialize(HasChildName constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(Children children, ConstraintValidatorContext constraintValidatorContext) {
```

Comment:
Why this validator doesn't add children to the context?

Reply:
I'm not quite sure what you mean by this Adam. Get me on slack and discuss.

Reply:
It's missing:
        constraintValidatorContext.buildConstraintViolationWithTemplate(constraintValidatorContext
	            .getDefaultConstraintMessageTemplate()).addPropertyNode("children").addConstraintViolation();

As other validators are doing, not required?

Reply:
Only required if I want to customise the property path. In this case, children (default) is fine.